### PR TITLE
Update new-onedrive-sync-client.md

### PR DESCRIPTION
**Pull request by BillSproule**
Updated with WNS and Telemetry opt-in recommendations. Clarified and corrected configuration details.

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 1]
```diff
@@ -20,13 +20,13 @@ description: "Learn how to configure the new OneDrive sync app (OneDrive.exe) fo
...
    
-When you deploy SharePoint Server 2019 in your organization, your users can sync their OneDrive files as well as SharePoint team site files by using the new OneDrive sync app (OneDrive.exe) for Windows. Compared with the previous OneDrive sync app (Groove.exe), the new sync app provides:
+When you deploy SharePoint Server 2019 in your organization, your users can sync their OneDrive files as well as SharePoint team site files by using the new OneDrive sync app (OneDrive.exe) for Windows or Mac. Compared with the previous OneDrive sync app (Groove.exe), the new sync app provides:
 - Improved performance and reliability
 - Files On-Demand
 - Support for larger files
 - Higher sync limits
-- The ability to silently deploy
-If your users are already syncing document libraries with the previous OneDrive sync app, they will transition to the new sync app automatically.
+- The ability to silently deploy.  If your Windows users are already syncing document libraries with the previous OneDrive sync app, they will transition to the new sync app automatically.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:28]:
```suggestion
- The ability to silently deploy. If your Windows users are already syncing document libraries with the previous OneDrive sync app, they will transition to the new sync app automatically.
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 2]
```diff
@@ -36,7 +36,28 @@ If your users are already syncing document libraries with the previous OneDrive
...
     - [Deploy OneDrive apps by using Intune](/onedrive/deploy-intune)
     - [Deploy and configure the new OneDrive sync app for Mac](/onedrive/deploy-and-configure-on-macos)
+    
+3. [Configure OneDrive for SharePoint Server 2019](new-onedrive-sync-client.md#Configure-OneDrive-for-SharePoint-Server-2019)
+  
+## Recommendations
+
+### 1. Allow WNS Traffic
+
+For the best user experience, SharePoint Server 2019 will send change notifications to sync clients via WNS web push notifications.  This feature ensures sync users quickly have the latest copies of any SharePoint Server file updates.  You may need to take steps to ensure outbound requests from your SharePoint Server and your user's sync client computers can reach and properly interact with the WNS service.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:46]:
```suggestion
For the best user experience, SharePoint Server 2019 will send change notifications to sync clients via WNS web push notifications. This feature ensures sync users quickly have the latest copies of any SharePoint Server file updates. You may need to take steps to ensure outbound requests from your SharePoint Server and your user's sync client computers can reach and properly interact with the WNS service.
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 3]
```diff
@@ -20,13 +20,13 @@ description: "Learn how to configure the new OneDrive sync app (OneDrive.exe) fo
 
 [!INCLUDE[appliesto-xxx-xxx-2019-xxx-md](../includes/appliesto-xxx-xxx-2019-xxx-md.md)]
    
-When you deploy SharePoint Server 2019 in your organization, your users can sync their OneDrive files as well as SharePoint team site files by using the new OneDrive sync app (OneDrive.exe) for Windows. Compared with the previous OneDrive sync app (Groove.exe), the new sync app provides:
+When you deploy SharePoint Server 2019 in your organization, your users can sync their OneDrive files as well as SharePoint team site files by using the new OneDrive sync app (OneDrive.exe) for Windows or Mac. Compared with the previous OneDrive sync app (Groove.exe), the new sync app provides:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:23]:
```suggestion
When you deploy SharePoint Server 2019 in your organization, your users can sync their OneDrive files as well as SharePoint team site files by using the new OneDrive sync app (OneDrive.exe) for Windows or Mac. Compared with the previous OneDrive sync app (Groove.exe), the new sync app provides:
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 4]
```diff
@@ -36,7 +36,28 @@ If your users are already syncing document libraries with the previous OneDrive
...
+3. [Configure OneDrive for SharePoint Server 2019](new-onedrive-sync-client.md#Configure-OneDrive-for-SharePoint-Server-2019)
+  
+## Recommendations
+
+### 1. Allow WNS Traffic
+
+For the best user experience, SharePoint Server 2019 will send change notifications to sync clients via WNS web push notifications.  This feature ensures sync users quickly have the latest copies of any SharePoint Server file updates.  You may need to take steps to ensure outbound requests from your SharePoint Server and your user's sync client computers can reach and properly interact with the WNS service.
+
+For SharePoint Server:
+  - Allow outbound HTTPS connection to reach \*.notify.windows.com
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:49]:
```suggestion
  - Allow outbound HTTPS connection to reach \*.notify.windows.com.
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 5]
```diff
@@ -36,7 +36,28 @@ If your users are already syncing document libraries with the previous OneDrive
...
   
+For OneDrive Sync client computers:
+  - Allow outbound TLS encrypted TCP/IP socket connection to reach \*.wns.windows.com
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:52]:
```suggestion
  - Allow outbound TLS encrypted TCP/IP socket connection to reach \*.wns.windows.com.
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 6]
```diff
@@ -36,7 +36,28 @@ If your users are already syncing document libraries with the previous OneDrive
...
+  - Avoid HTTPS decryption for \*.wns.windows.com on your proxy server as this likley will disconnect the socket connection
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:53]:
```suggestion
  - Avoid HTTPS decryption for \*.wns.windows.com on your proxy server as this likely will disconnect the socket connection.
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 7]
```diff
@@ -36,7 +36,28 @@ If your users are already syncing document libraries with the previous OneDrive
...
+  
+If either the SharePoint Server or the OneDrive sync client is unable to communicate with the WNS service then the sync client will fall back to polling the SharePoint Server roughly every 2 minutes looking for changes.  As a result your users may see delays of more than 2 minutes from the time of a server change to when the sync client downloads the changed file.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:55]:
```suggestion
If either the SharePoint Server or the OneDrive sync client is unable to communicate with the WNS service then the sync client will fall back to polling the SharePoint Server roughly every two minutes looking for changes.  As a result, your users may see delays of more than two minutes from the time of a server change to when the sync client downloads the changed file.
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 8]
```diff
@@ -52,12 +73,12 @@ Configure the following two Group Policy objects to configure OneDrive to be use
...
 
 **Specify the OneDrive location in a hybrid environment**
 
-This setting lets you specify if the sync app should authenticate against SharePoint in Microsoft 365 or the SharePoint on-premises server if the identity exists in both identity providers.
+This setting (SharePointOnPremPrioritization) lets you specify if the sync app should first set up a sync relationship with SharePoint in Microsoft 365 (the default) or the SharePoint on-premises server if the user identity exists in both identity providers. The sync application's Settings dialog can be used to "Add Account" the same identity for the other SharePoint realm after the first has been configured (if the user identity exists in both).
 
 You should be able to find these Group Policy objects using the Group Policy Editor (gpedit.msc) when navigating to Computer Configuration\Administrative Templates\OneDrive. If the OneDrive folder is not present, you can add the OneDrive Group Policy template by coping the following two files from the OneDrive installation folder after you have installed the latest OneDrive sync app on that computer:
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:83]:
```suggestion
You should be able to find these Group Policy objects using the Group Policy Editor (gpedit.msc) when navigating to Computer Configuration\Administrative Templates\OneDrive. If the OneDrive folder is not present, you can add the OneDrive Group Policy template by copying the following two files from the OneDrive installation folder after you have installed the latest OneDrive sync app on that computer:
```

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 9]
```diff
@@ -82,20 +103,20 @@ More information:
...
 
 If your organization also uses the OneDrive sync app to sync files in Microsoft 365, here's what will be different for users who sync on-premises files.
 
 ### Single Top-level URL
-The OneDrive sync client only allows synchronization with a single URL (including all Document Libraries in all Site Collections and subsites of the specified URL). It is not possible to add additional top-level URLs (Web Applications or Host-Named Site Collections) for the OneDrive synchronization client with SharePoint Server 2019.
+If you have deployed multiple on premises SharePoint Server farms in your enterprise, on a given client computer, you will only specify a single "SharePointOnPremFrontDoorUrl".  For a given user, you should configure their computer with the SharePoint Server URL that hosts their individual OneDrive site or if they don't have a OneDrive site, where the Team Sites they are most likely to sync are hosted.  Your users will be able to start syncing Team Site content from any of your on premises SharePoint Server farms by navigating to the web experience of the Team Site and clicking the Sync button on that site.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:117]:
```suggestion
If you have deployed multiple on-premises SharePoint Server farms in your enterprise, on a given client computer, you will only specify a single "SharePointOnPremFrontDoorUrl". For a given user, you should configure their computer with the SharePoint Server URL that hosts their individual OneDrive site or if they don't have a OneDrive site, where the team sites they are most likely to sync are hosted. Your users will be able to start syncing team site content from any of your on-premises SharePoint Server farms by navigating to the web experience of the Team Site and clicking the Sync button on that site.
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointServer/install/new-onedrive-sync-client.md:117]: Please remove extra spaces after periods, hyphenate all instances of "on-premises", and lower case all instances of "team site".

File: SharePoint/SharePointServer/install/new-onedrive-sync-client.md [hunk 10]
```diff
@@ -82,20 +103,20 @@ More information:
...
 
-In SharePoint in Microsoft 365, you may synchronize with multiple top-level URLs, such as https://contoso-my.sharepoint.com and https://contoso.sharepoint.com.
+For example you have a SharePoint Server farm for your Finance department and another for the rest of your company. Users who are members of the Finance department have their individual OneDrive site hosted on the Fianance department's SharePoint Server farm.  For those Finance employees, you use your computer management system to set the SharePointOnPremFrontDoorUrl registry key policy on their computer to have your Finance specific farm's URL.  For all other employees you set the SharePointOnPremFrontDoorUrl to your other SharePoint Server farm URL.  The sync client will look for and provision on demand as needed, the user's OneDrive site on the appropriate SharePoint Server farm.
```
Comment (JohanFreelancer9) [inline SharePoint/SharePointServer/install/new-onedrive-sync-client.md:119]:
```suggestion
For example, you have a SharePoint Server farm for your Finance department and another for the rest of your company. Users who are members of the Finance department have their individual OneDrive site hosted on the Finance department's SharePoint Server farm.  For those Finance employees, you use your computer management system to set the SharePointOnPremFrontDoorUrl registry key policy on their computer to have your Finance specific farm's URL. For all other employees, you set the SharePointOnPremFrontDoorUrl to your other SharePoint Server farm URL.  The sync client will look for and provision on-demand as needed, the user's OneDrive site on the appropriate SharePoint Server farm.
```

Comment (JohanFreelancer9) [reply SharePoint/SharePointServer/install/new-onedrive-sync-client.md:119]: Please add commas as indicated, remove extra spaces after periods, correct typo (Fianance), and hyphenate on-demand.

**Comment by BillSproule** [comment]
Thanks Johan for all your improvements.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**